app/src/main/java/org/oppia/android/app/player/state/ImageRegionSelectionInteractionView.kt

```diff
@@ -127,7 +127,7 @@ class ImageRegionSelectionInteractionView @JvmOverloads constructor(
       )
       areasImage.addRegionViews()
       if (::lastSelectedRegion.isInitialized && lastSelectedRegion.hasRegion()) {
-        areasImage.toggleRegion(lastSelectedRegion)
+        areasImage.toggleRegion(lastSelectedRegion, null)
```

Comment:
The ``null`` should include a named parameter to provide context, e.g. ``paramName = null``.

Reply:
Done